src/components/network/network-map.js

```diff
@@ -187,6 +187,7 @@ const NetworkMap = forwardRef((props, ref) => {
             network: props.network,
             geoData: props.geoData,
             getNominalVoltageColor: getNominalVoltageColor,
+            disconnectedLineColor: labelColor,
```

Comment:
can you either rename the labelColor variable ( "foregroundNeutralColor" ? )
or create a separate variable disconnectedLineColor (with a separate computation, even if it's the same value as the the labelColor ) ?